src/EppoClient.php

```diff
@@ -181,9 +223,18 @@ public function getParsedJSONAssignment(string $subjectKey, string $flagKey, arr
      * @throws InvalidArgumentException
      * @throws SimpleCacheInvalidArgumentException
      */
-    public function getJSONStringAssignment(string $subjectKey, string $flagKey, array $subjectAttributes = []): string {
-        $parsedJsonValue = $this->getParsedJSONAssignment($subjectKey, $flagKey, $subjectAttributes);
-        return isset($parsedJsonValue) ? json_encode($parsedJsonValue) : null;
+    public function getJSONStringAssignment(string $subjectKey, string $flagKey, array $subjectAttributes = []): ?string
```

Comment:
fyi @aarsilv I found we were returning `string` here; in php returning `null` is not allowed so I changed to `?string` - this also matches the other functions.

bumping to 1.3.0 as its a minor api change.

Reply:
oh good catch!